content/en/blog/_posts/2021-10-31-kubernetes-in-kubernetes-and-pxe-bootable-servers-farm.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: blog
+title: "Kubernetes-in-Kubernetes and PXE bootable servers farm"
```

Comment:
I don't know what the Russian original wording was, so please comment if this is no longer faithful, but I think I'd write:
```suggestion
title: "Kubernetes-in-Kubernetes and our PXE bootable server farm"
slug: kubernetes-in-kubernetes-and-pxe-bootable-server-farm
```

Reply:
I think your suggestion is more correct here, so I'm going to accept it.